apps/condo/domains/common/components/PhoneLink.tsx

```diff
@@ -8,6 +7,6 @@ interface IPhoneLink {
 
 export const PhoneLink: React.FC<IPhoneLink> = ({ value }) => (
     <a href={`tel:${normalizePhone(value)}`}>
-        {formatPhone(value)}
+        {value}
```

Comment:
why we don't need to format the phone?

Reply:
30+ countries with unique phone pattern. It's easier to disable phone formatting.